packages/specifications/src/candidature/candidature.world.ts

```diff
@@ -91,7 +91,7 @@ export class CandidatureWorld {
       dateÉchéanceGf: expectedValues.dateÉchéanceGfValue
         ? DateTime.convertirEnValueType(expectedValues.dateÉchéanceGfValue)
         : undefined,
-      emailContact: expectedValues.emailContactValue,
+      emailContact: Email.convertirEnValueType(expectedValues.emailContactValue.toLowerCase()),
```

Comment:
Comme ça ici t'as pas à passer en minuscule pour comparer


```suggestion
      emailContact: Email.convertirEnValueType(expectedValues.emailContactValue),
```